lib/galaxy/exceptions/__init__.py

```diff
@@ -64,7 +64,7 @@ class AcceptedRetryLater(MessageException):
     err_code = error_codes_by_name["ACCEPTED_RETRY_LATER"]
     retry_after: int
 
-    def __init__(self, msg, retry_after=60):
+    def __init__(self, msg: str, retry_after=60):
```

Comment:
I think the convention is that the msg attribute is optional ? If you do make the suggested change in https://github.com/galaxyproject/galaxy/pull/18041/files#r1578083608 this would likely be a type error since it doesn't match the super class signature (but I wouldn't go so far as to say I understand when mypy flags type errors)

```suggestion
    def __init__(self, msg: Optional[str] = None, retry_after=60):
```